### ISSUE TITLE
Tab stops are not accessible, different browsers showing different results

### ISSUE THREAD

**Issue by awentzel** [microsoft/fast#3978]
**Describe the bug; what happened?**
There is no indication the users focus state has landed on certain controls.

1. On MacOS -> Edge, social medial icons on https://www.fast.design.
2. On MacOS -> Safari/Firefox tabbing jumps to the center of the page.

**What are the steps to reproduce the issue?**
1. Visit the https://www.fast.design and start tabbing on Edge.
2. Visit the https://www.fast.design and start tabbing on Firefox.
3. Visit the https://www.fast.design and start tabbing on Safari.

**What behavior did you expect?**
Items should include decoration to indicate tab focus and be in the correct order starting at the top of page and working down to the bottom.

**Comment by radium-v** [comment]
This is expected behavior on MacOS - https://support.apple.com/en-us/HT204434#fullkeyboard

**Comment by chrisdholt** [comment]
Per @radium-v there are a couple ways MacOS allows you to navigate. This also could be an issue with a new OS - what version are you on @awentzel? I do not repro this on Cataline 10.15.5.

**Comment by CuddleBunny** [comment]
Firefox on Windows has tab stops on both the custom element and the internal input as well. If you have two `fast-text-field` elements in a row it will take two tab presses to move focus from the first to the second.

**Comment by chrisdholt** [comment]
@CuddleBunny is this specific to fast-text-field or are there multiple component instances where you're seeing this? What you are describing seems more linked to https://github.com/microsoft/fast/issues/3882 than the above IMO.

**Comment by CuddleBunny** [comment]
@chrisdholt - actually it looks like it was fixed in the most recent release. Ignore me 🙃
